src/dev.sh

```diff
@@ -169,21 +157,23 @@ function package ()
         echo "You must build first.  Expecting to find ${LAYOUT_DIR}/bin"
     fi
 
-    agent_ver=`${LAYOUT_DIR}/bin/Agent.Listener --version` || failed "version"
+    agent_ver=$("${LAYOUT_DIR}/bin/Agent.Listener" --version) || failed "version"
     agent_pkg_name="vsts-agent-${RUNTIME_ID}-${agent_ver}"
 
     heading "Packaging ${agent_pkg_name}"
 
-    rm -Rf ${LAYOUT_DIR}/_diag
-    find ${LAYOUT_DIR}/bin -type f -name '*.pdb' -delete
-    mkdir -p $PACKAGE_DIR
-    pushd $PACKAGE_DIR > /dev/null
-    rm -Rf *
+    rm -Rf "${LAYOUT_DIR:?}/_diag"
+    find "${LAYOUT_DIR}/bin" -type f -name '*.pdb' -delete
+
+    mkdir -p "$PACKAGE_DIR"
+    rm -Rf "${PACKAGE_DIR:?}/*"
```

Comment:
The * here got caught in the quotes.

Reply:
👍 good catch, you want to create a PR? :)

Reply:
Thanks for the encouragement, but I'm afraid I care only enough to drop a drive-by comment. You can decide if it's actually worth fixing. :)